utilities/BuildUtilities.groovy

```diff
@@ -331,7 +331,7 @@ def relativizeFolderPath(String folder, String path) {
  * getScannerInstantiates - returns the mapped scanner or default scanner
  */
 def getScanner(String buildFile){
-	if (props.runzTests == "True") {
+	if (props.runzTests) {
```

Comment:
It is enough ? Or should we have this ?
if (props.runzTests && props.runzTests .toBoolean())